README.md

```diff
@@ -49,20 +48,23 @@ Or install it yourself as:
     # config/initializers/omniauth.rb
 
     Rails.application.config.middleware.use OmniAuth::Builder do
-      provider :census, "CENSUS_ID", "CENSUS_SECRET", {
+      provider :census, ENV["CENSUS_ID"], ENV["CENSUS_SECRET"], {
```

Comment:
Def an improvement! Perhaps add a note the the application can call these variable whatever it wants.